packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -51,7 +52,8 @@ async function createViewRemoteVideo(
     name: EventNames.CREATING_REMOTE_VIEW,
     level: 'info',
     message: 'Start creating view for remote video.',
-    data: streamLogInfo
+    data: streamLogInfo,
+    streamType: streamType
```

Comment:
Given this is extra metadata, move this into the streamLogInfo object (i.e. part of the `data` field of the logEvent)